arm-trafficmanager/2015-11-01/trafficmanager.json

```diff
@@ -0,0 +1,699 @@
+      },
+      "patch": {
+        "tags": [
+          "Profiles"
+        ],
+        "operationId": "Profiles_Update",
+        "description": "Update a Traffic Manager profile.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager profile."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            },
+            "description": "The Traffic Manager profile parameters supplied to the Update operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": { 
+             "description": "", 
+             "schema": { 
+               "$ref": "#/definitions/Profile" 
+             } 
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Endpoint": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the ID of the Traffic Manager endpoint."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the Traffic Manager endpoint."
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets the endpoint type of the Traffic Manager endpoint."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "targetResourceId": {
+            "type": "string",
+            "description": "Gets or sets the Azure Resource URI of the of the endpoint.  Not applicable to endpoints of type 'ExternalEndpoints'."
+		      },
+          "target": {
+            "type": "string",
+            "description": "Gets or sets the fully-qualified DNS name of the endpoint.  Traffic Manager returns this value in DNS responses to direct traffic to this endpoint."
+          },
+          "endpointStatus": {
+            "type": "string",
+            "description": "Gets or sets the status of the endpoint..  If the endpoint is Enabled, it is probed for endpoint health and is included in the traffic routing method.  Possible values are 'Enabled' and 'Disabled'."
+          },
+          "weight": {
+            "type": "integer",
+            "format": "int64",
+            "description": "Gets or sets the weight of this endpoint when using the 'Weighted' traffic routing method. Possible values are from 1 to 1000."
+          },
```

Comment:
You can add constraints like [exclusive/inclusive][Max/Min]. Take a look over here: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#patterned-fields-2. Scroll a little upwards and you should see the constraints.